### DISCUSSION TITLE
how to run github in google colab

### DISCUSSION THREAD

**Discussion by 1069as** [community/community#43287] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Question

### Body

https://github.com/MoonBlvd/tad-IROS2019.git
this is the github link i would like to run yesterday, but in google colab it can't find the path. how do i solve it?

**Comment by jfullstackdev** [comment]
maybe this will help 

https://medium.com/analytics-vidhya/how-to-use-google-colab-with-github-via-google-drive-68efb23a42d

**Comment by mishra-prajjawal** [comment, marked as answer]
Hi 👋 @1069as  Thanks for reaching out with this problem of yours.

Answer to that is just click on the left to the files icon and then navigate yourself to the content directory.  

Aa of downloading assets part use wget 
Type in the code box and run the particular box in Google Colab Notebook 
` !wget https://github.com/MoonBlvd/tad-IROS2019.git `

**Comment by Ankush11903** [comment]
Please let us know if there is still an issue or if it is resolved then mark this as answered.

**Comment by 1069as** [comment]
okey thank you guy, but now do you guys have ever seen a project that related to Detecting Abnormal Event 
in Traffic Scenes using Unsupervised Deep Learning Approach.
